### DISCUSSION TITLE
Improving the experience for people interested in ai16z and Eliza

### DISCUSSION THREAD

**Discussion by Johnwbusch** [elizaOS/eliza#1491] [Ideas]
I see the vision and believe in the mission of ai16z and the Eliza framework that @shawmakesmagic shared during the Bankless Podcast. I want to help grow and broaden the community by improving the experience for people who are interested in ai16z and being a holder, and possibly being part of the community, even though they are not savvy when it comes to AI and Crypto.

My ai16z journey began with hearing about ai16z during this Bankless podcast https://youtu.be/5GBXS5myXz0?si=FexO-UAF7yF2y-9i

After enjoying the entire podcast, I went to Kraken to buy ai16z tokens, unfortunately, Kraken doesn't offer ai16z tokens yet, so I went to Coin Market Cap to see the Exchanges where I could buy ai16z tokens

Below are 3 recommendations to improve the experience for people who are interested in ai16z 

Recommended improvement 1: someone at ai16Z contact Coin Market Cap and ask them to change the hyperlinked URL within https://coinmarketcap.com/currencies/ai16z/ from what it currently is https://ai16z.ai/ to https://github.com/elizaOS/eliza  I created a post in the CMC Community https://coinmarketcap.com/community/ (see attached screenshot)
![Screenshot 2024-12-26 at 9 15 14 PM](https://github.com/user-attachments/assets/5a024d65-eafe-4484-85f5-8fd9f1f0658e)

Recommended improvement 2: Create a page on Git Hub with Basic Instructions for US-based users about how to buy ai16Z with the Raydium or Orca DEX, a Phantom wallet, and a Ledger. Then get this page showing in the top five results when a user googles ai16z.

Recommended improvement 3: someone at ai16Z please contact Coinbase and ask them to remove the 3 tokens in the search result for the REAL ai16z within the Coinbase Wallet. Fortunately for me, I knew to tap on the link and check the Token address, yet many people interested in the REAL ai16z do not know to check the token address and they will ape in and get ripped off buying a token that is not the REAL ai16z.
![ai16z in Coinbase Wallet](https://github.com/user-attachments/assets/bfd06e8a-b251-4f09-882f-4d10a771dd31)

**Comment by zbailey83** [comment]
Nice post.

Unfortunately, due to their bias towards BASE, Coinbase is highly unlikely to do anything in order to help a Solana SPL token, like $ai16z

If it was on BASE they'd probably create a Coinbase Learn campaign on it.
